.dev/CRAN_Release.cmd

```diff
@@ -114,7 +114,7 @@ grep asCharacter *.c | grep -v PROTECT | grep -v SET_VECTOR_ELT | grep -v setAtt
 
 cd ..
 R
-cc(clean=TRUE)  # to compile with -pedandic -Wall -flto. Also uses very latest gcc (currently gcc-7) as CRAN does
+cc(clean=TRUE, CC="gcc-9")  # to compile with -pedandic -Wall. Use recent gcc as CRAN does
```

Comment:
maybe add a link here for easier look-up?

https://cran.r-project.org/web/checks/check_flavors.html

Reply:
Good idea: idea.